nessie/src/main/java/org/apache/iceberg/nessie/NessieIcebergClient.java

```diff
@@ -46,15 +50,7 @@
 import org.projectnessie.error.NessieNamespaceNotFoundException;
 import org.projectnessie.error.NessieNotFoundException;
 import org.projectnessie.error.NessieReferenceNotFoundException;
-import org.projectnessie.model.Branch;
-import org.projectnessie.model.Content;
-import org.projectnessie.model.ContentKey;
-import org.projectnessie.model.EntriesResponse;
-import org.projectnessie.model.GetNamespacesResponse;
-import org.projectnessie.model.IcebergTable;
-import org.projectnessie.model.Operation;
-import org.projectnessie.model.Reference;
-import org.projectnessie.model.Tag;
+import org.projectnessie.model.*;
```

Comment:
I don't think we allow * imports in the project

Reply:
It is draft PR. My bad. Let me clean it up.